zap/src/main/java/org/zaproxy/zap/extension/api/RustAPIGenerator.java

```diff
@@ -93,8 +93,7 @@ private void generateRustElement(ApiElement element, String component, String ty
             out.write("/**\n");
             out.write(" * " + desc + "\n");
             if (isOptional()) {
-                out.write(" * <p>\n");
-                out.write(" * " + OPTIONAL_MESSAGE + "\n");
+                out.write(OPTIONAL_MESSAGE + "\n");
```

Comment:
Was this changed on purpose? It generates the following now:
```rust
/**
 * Endpoint description.
This component is optional and therefore the API will only work if it is installed
*/
```
I'd expect the ` * ` before the optional message.

Reply:
Also, the other newline would help separate the optional message from the description.

Reply:
I agree it is missing the " * " in front of the optional message